src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+const cellsLength = table.rows[0].cells.length;
+
+[...rows].forEach((row) => {
+  const secondRowCopy = row.cells[1].cloneNode(true);
+
+  row.cells[cellsLength - 1].before(secondRowCopy);
+});
```

Comment:
Ensure that each row has at least two cells before attempting to clone the second cell. You can add a condition to check if `row.cells.length >= 2` to prevent potential errors when the table has rows with fewer than two cells.